website/content/docs/k8s/connect/index.mdx

```diff
@@ -345,16 +337,13 @@ Installation of the mutating admission webhook is automated using the
 To install the Connect injector, enable the Connect injection feature using
 [Helm values](/docs/k8s/helm#configuration-values) and
 upgrade the installation using `helm upgrade` for existing installs or
-`helm install` for a fresh install. The Connect injector **also requires**
```

Comment:
client agents and grpc are enabled by default and helm will give an error if you try to use connect and disable them